### DISCUSSION TITLE
Multiple Flows on one Page

### DISCUSSION THREAD

**Discussion by JanLoehr** [xyflow/xyflow#986] [Q&A] [answered]
Hey, great library!

Is it me, or can't you have to Flows on one Page?

I'm trying to achieve something like this.
![image](https://user-images.githubusercontent.com/11177594/110858233-1dc66000-82ba-11eb-8db7-cbd5a7f70b90.png)

But on both sides the outputs should be pointing to the center. I've got separate element arrays defined and feed them into two Flows. But both always show the same elements.

If I get rid of the "other one" both display correctly.
![image](https://user-images.githubusercontent.com/11177594/110858592-8ca3b900-82ba-11eb-9160-63d39f15c80e.png)
![image](https://user-images.githubusercontent.com/11177594/110858618-988f7b00-82ba-11eb-956d-7cb4e12014e9.png)

So I assume, this isn't possible?

**Comment by leejustin** [comment, marked as answer]
You need multiple providers for multiple flows: https://reactflow.dev/docs/api/components/provider/

**Reply by JanLoehr** [reply]
Oohhh, awesome. Missed that. Thanks a lot!!
